.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,72 @@
+name: General Question
```

Comment:
Do you want to receive these here, or in the forum? Generally speaking I'd suggest general questions or support go via the Docker forum: https://forum.mautic.org/c/support/docker-support/112 - as this is what we do with mautic/mautic and that way it keeps the issue queue clear for issues. The forum is also very well indexed in search engines. Totally up to you though!